Functions.Rmd

```diff
@@ -697,7 +697,7 @@ sample <- function(x, size = NULL, replace = FALSE, prob = NULL) {
 }
 ```
 
-You can make that pattern even simpler with a small helper. The infix `%||%` function uses the LHS if it's not null, otherwise it uses the RHS:
+You can simplify this task with the help of the infix `%||%` function, which uses the LHS if it's not `NULL` and the RHS otherwise:
```

Comment:
I think pattern is important here

Reply:
The pattern is `argument %||% default_value`, and you might use that pattern in many different functions.